contrib/format.sh

```diff
@@ -21,7 +21,7 @@ fi
 
 cd "$(dirname $0)/../"
 if [ "$1" = "verify" ] ; then
-    if [ $($binary --output-replacements-xml $(find jni daemon llarp include pybind | grep -E '\.([hc](pp)?|mm?)$' | grep -v '\#') | grep '</replacement>' | wc -l) -ne 0 ] ; then
+    if [ $($binary --output-replacements-xml $(find jni daemon llarp include pybind | grep -E '\.([hc](pp)?|mm)$' | grep -v '\#') | grep '</replacement>' | wc -l) -ne 0 ] ; then
```

Comment:
Why don't we just let it be reformatted, by adding the `?` into the pattern 4 lines down?

Reply:
it then requires an almost full reformat of the existing .m compilation units

Reply:
Good.